Hunting Queries/SigninLogs/UserAccounts-NewSingleFactorAuth.yaml

```diff
@@ -18,8 +18,8 @@ tags:
   - AADSecOpsGuide
 query: |
 
-  let starttime = todatetime('{{StartTimeISO}}');
-  let endtime = todatetime('{{EndTimeISO}}');

```

Comment:
this is giving error while running. any reason for removing 'todatetime'?

Reply:
The KQL validation test is looking for timespan type value for these two variables, I have updated the function syntax 'totimespan' now and it is running without error. Thanks!